lib/packer/builders/amazon.rb

```diff
@@ -12,7 +12,7 @@ def initialize
           'instance_type',
           'region',
           'source_ami',
-          'ssh_username'
+          ['ssh_username', 'communicator']
```

Comment:
The syntax here isn't correct, but don't worry about it -- I'll fix it up before I do a release. I'll send you the commit so you can see the correct syntax. I'm going to leave `communicator` off the list as it's optional in all the builders in Packer right now.